src/try_from_row.rs

```diff
@@ -1,14 +1,57 @@
-use common::get_struct_fields;
+use common::{struct_fields, tokenize_fields, get_ident, has_attr};
 use quote;
 use syn;
 
 pub fn impl_try_from_row(ast: &syn::DeriveInput) -> quote::Tokens {
   let name = &ast.ident;
-  let fields = get_struct_fields(ast);
+  let fields = struct_fields(ast).clone();
+  let (mapped, non_mapped): (Vec<syn::Field>, Vec<syn::Field>) = fields
+      .into_iter()
+      .partition(|r| has_attr(r, "json_mapped"));
+  let mut fields = tokenize_fields(&non_mapped);
+
+  for mapped in mapped {
+    let name = mapped.ident.unwrap();
+    let ty = get_ident(mapped.ty);
+
+    let string = quote! {
+          let val = &String::from_cdrs_r(&cdrs, stringify!(#name))?;
+      };
+
+    let serde = quote! {
+          serde_json::from_str(&val).map_err(|e| cdrs_tokio::Error::General(format!("Failed to transform type {}", stringify!(#name))))?
```

Comment:
We should add serde dependency.

Reply:
Edit: typo - we should NOT add serde dependency.

Reply:
serde does not need to be added, as long as users don't annotate fields with json_mapped. If they do, they need to add serde in their own project.

Reply:
Well, this code doesn't handle non-json fields at the moment. Also, where does `json_mapped` attribute come from? It's not in `serde` which uses `#[serde(skip)]` to skip fields.

Reply:
json_mapped is from the CDRS_ORM crate